org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporter.java

```diff
@@ -100,37 +104,29 @@ private void importDir(Path rootFolder, IProgressMonitor monitor) {
 				}
 			}
 		}
-		startSynchronization(rootFolder.toFile(), distribution, NewProjectHandler.IMPORT_AND_MERGE);
+		return distribution;
 	}
 
-	protected void startSynchronization(File location, GradleDistribution distribution, NewProjectHandler newProjectHandler) {
-		List<String> jvmArgs = new ArrayList<>();
-		IProxyService proxyService = JavaLanguageServerPlugin.getInstance().getProxyService();
-		IProxyData proxy = proxyService.getProxyData(IProxyData.HTTP_PROXY_TYPE);
-		if (proxy != null) {
-			addArgs(jvmArgs, JavaLanguageServerPlugin.HTTP_PROXY_HOST, proxy.getHost());
-			if (proxy.getPort() > 0) {
-				addArgs(jvmArgs, JavaLanguageServerPlugin.HTTP_PROXY_PORT, String.valueOf(proxy.getPort()));
-			}
-			addArgs(jvmArgs, JavaLanguageServerPlugin.HTTP_PROXY_USER, proxy.getUserId());
-			addArgs(jvmArgs, JavaLanguageServerPlugin.HTTP_PROXY_PASSWORD, proxy.getPassword());
-		}
-		proxy = proxyService.getProxyData(IProxyData.HTTPS_PROXY_TYPE);
-		if (proxy != null) {
-			addArgs(jvmArgs, JavaLanguageServerPlugin.HTTPS_PROXY_HOST, proxy.getHost());
-			if (proxy.getPort() > 0) {
-				addArgs(jvmArgs, JavaLanguageServerPlugin.HTTPS_PROXY_PORT, String.valueOf(proxy.getPort()));
```

Comment:
why is the proxy stuff deleted here?

Reply:
jvmArgs are ignored. Buildship uses the Eclipse proxy settings. See https://github.com/eclipse/buildship/blob/master/org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/workspace/internal/DefaultGradleWorkspaceManager.java#L49

Reply:
ok, have you checked against a proxy this still works?

Reply:
jvmArgs was always ignored.